<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="qmlWidget" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    background-color: #f0f0f0;               /* Светлый фон */
    border: 1px solid #696969;               /* Светло-серая граница */
    border-radius: 10px;                     /* Закругленные края */
    text-align: center;                      /* Центрирование текста */
    font-size: 12px;                         /* Размер шрифта */
    color: #555555;                          /* Темный цвет текста */
    min-height: 25px;                        /* Высота прогресс-бара */
}

QProgressBar::chunk {
    background-color: qlineargradient(
        spread:pad, x1:0, y1:0, x2:1, y2:1,
        stop:0 #d0d0d0, stop:1 #696969);     /* Матовый серый градиент */
    border-radius: 8px;                      /* Закругленные края сегмента */
    margin: 2px;                             /* Отступы от краев */
}
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="format">
     <string>%v Amp</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar22">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    background-color: #f0f0f0;               /* Светлый фон */
    border: 1px solid #696969;               /* Светло-серая граница */
    border-radius: 10px;                     /* Закругленные края */
    text-align: center;                      /* Центрирование текста */
    font-size: 12px;                         /* Размер шрифта */
    color: #555555;                          /* Темный цвет текста */
    min-height: 25px;                        /* Высота прогресс-бара */
}

QProgressBar::chunk {
    background-color: qlineargradient(
        spread:pad, x1:0, y1:0, x2:1, y2:1,
        stop:0 #d0d0d0, stop:1 #696969);     /* Матовый серый градиент */
    border-radius: 8px;                      /* Закругленные края сегмента */
    margin: 2px;                             /* Отступы от краев */
}
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="format">
     <string>%v Temp</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStatistic">
    <property name="title">
     <string>Statistic</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menuConnection_type">
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="title">
     <string>Connection type</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionrtu"/>
   </widget>
   <widget class="QMenu" name="menustatistictwo">
    <property name="title">
     <string>statistictwo</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menuModbusManual">
    <property name="title">
     <string>ModbusManual</string>
    </property>
    <addaction name="actionOpenManual"/>
   </widget>
   <addaction name="menuStatistic"/>
   <addaction name="menustatistictwo"/>
   <addaction name="menuConnection_type"/>
   <addaction name="menuModbusManual"/>
  </widget>
  <action name="actionrtu">
   <property name="text">
    <string>вариант подключения</string>
   </property>
   <property name="toolTip">
    <string>настройки подключения</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Открыть окно статистики</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>статистика</string>
   </property>
  </action>
  <action name="actionOpenManual">
   <property name="text">
    <string>Open Manual</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>menubar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>MainWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
